src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends FigureTwo implements AreaOfFigure,InfoFigure {
+    //квадрат
+
+    public Square(String color, double firsSide) {
+        super(color, firsSide, firsSide);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return getSecondSide() * getFirsSide();
```

Comment:
There is a typo in the method name `getFirsSide`. It should be `getFirstSide` to match the corrected variable name in the `FigureTwo` class.